meetup/meetup.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Write your code for the 'Meetup' exercise in this file. Make the tests in
+`meetup_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/meetup` directory.
+=end
+
+require 'date'
+
+WEEKDAYS = [:sunday, :monday, :tuesday, :wednesday, :thursday, :friday, :saturday]
+ORDINALS = [:first, :second, :third, :fourth, :fifth] 
```

Comment:
```suggestion
WEEKDAYS = { sunday: 0, monday: 1, tuesday: 2, wednesday: 3, thursday: 4, friday: 5, saturday: 6 }
ORDINALS = { first: 0, second: 1, third: 2, fourth: 3, fifth: 4 } 
Hash is more preferable here, because search in the array takes n complexity, while search in the hash takes 1.
Also when you use array in that way you are dependent on the sequence of the elements, and that is not desired
```